language/resource.language.en_gb/strings.po

```diff
@@ -113,7 +113,7 @@ msgid "Send boot config files from /flash, kodi_crash.log, kernel log and Samba
 msgstr ""
 
 msgctxt "#720"
-msgid "Set to ON and Bluetooth devices will be disabled during power-saving modes"
+msgid "Set to ON to enable the Bluetooth (BlueZ) service"
```

Comment:
```suggestion
msgid "Set to ON to enable the Bluetooth service"
```
I'm not sure that mentioning `BlueZ` (which won't mean anything to most people) helps improve this message, so it might be best to remove it.

Reply:
Done